### PR TITLE
Add list selector to TodoEditor(for #14)

### PR DESCRIPTION
**Pull request by AnubhaAgrawal**
Now, it support multiple vdirs.

### CODE REVIEW

**Comment by untitaker** [comment]
Hello @AnubhaAgrawal and thanks for your PR! However, I think your changes will crash the editor. I think I gave your team partner a few resources to get started with setting up todoman from a user perspective. If you use `todo edit`, you'll see what I mean.

Also your pull requests contains some changes that you probably didn't mean to make (see "Files" tab). In particular it reverts some recent changes made to the master branch.

**Comment by AnubhaAgrawal** [comment]
Can u please tell that the function of **Radiobutton** that I made is correct or not.
Now, I  know that the pp( ) function have to be removed and also it required some changes. but firstly I want to know that the **Radiobutton** function that I made is correct or not

File: todoman/ui.py [hunk 1]
```diff
@@ -24,19 +24,23 @@ class TodoEditor:
     The UI for a single todo entry.
     """
 
-    def __init__(self, todo, lists, formatter):
+    def __init__(self, todo, lists, formatter,current_list):
```
Comment (hobarrera) [inline todoman/ui.py]: No `current list` is being passed when creating new `TodoEditor` instances (so this crashes).

I think you didn't mean to keep this bit.

File: todoman/ui.py [hunk 2]
```diff
@@ -24,19 +24,23 @@ class TodoEditor:
...
         self.todo = todo
-        self.lists = lists
+        self.lists = list(lists)
         self.formatter = formatter
         self.saved = EditState.none
         self._loop = None
 
         self._msg_text = urwid.Text('')
 
+        urwid.RadioButton(self.lists, "label", state='first True', on_state_change=self.current_list+1, user_data=None)
```
Comment (hobarrera) [inline todoman/ui.py]: Not sure what you meant to do here by `+1`; `current_list` is a `List`, so you can't add +1 to it.

File: todoman/ui.py [hunk 3]
```diff
@@ -24,19 +24,23 @@ class TodoEditor:
...
+    def pp(self):
```
Comment (hobarrera) [inline todoman/ui.py]: This cuts `__init__` in half, so initialization crashes.

**Comment by untitaker** [comment]
You are creating a `RadioButton` but don't save it anywhere. Also you are adding `1` to a `List` object, but this operation isn't defined and crashes.

**Comment by AnubhaAgrawal** [comment]
```(venv_todo) anubha@anubha-Inspiron-3542:~/todoman/todoman$ tox -epy3
GLOB sdist-make: /home/anubha/todoman/setup.py
py3 inst-nodeps: /home/anubha/todoman/.tox/dist/todoman-2.1.0.post20+ng6e11902.d20170223.zip
py3 installed: appdirs==1.4.1,atomicwrites==1.1.5,click==6.7,click-log==0.1.8,configobj==5.0.6,coverage==4.3.4,flake8==3.3.0,flake8-import-order==0.12,freezegun==0.3.8,future==0.16.0,hypothesis==3.6.1,icalendar==3.11.3,mccabe==0.6.1,packaging==16.8,parsedatetime==2.2,py==1.4.32,pycodestyle==2.3.1,pyflakes==1.5.0,pyparsing==2.1.10,pytest==3.0.6,pytest-cov==2.4.0,python-dateutil==2.6.0,pytz==2016.10,pyxdg==0.25,six==1.10.0,todoman==2.1.0.post20+ng6e11902.d20170223,urwid==1.3.1
py3 runtests: PYTHONHASHSEED='2643296417'
py3 runtests: commands[0] | py.test --cov todoman
============================================================= test session starts ==============================================================
platform linux -- Python 3.5.2, pytest-3.0.6, py-1.4.32, pluggy-0.4.0
rootdir: /home/anubha/todoman, inifile: tox.ini
plugins: cov-2.4.0, hypothesis-3.6.1
collected 50 items 

tests/test_basic.py ......x.................
tests/test_config.py .......xx.
tests/test_filtering.py ........
tests/test_formatter.py F
tests/test_main.py .
tests/test_model.py ...
tests/test_porcelain.py ...

----------- coverage: platform linux, python 3.5.2-final-0 -----------
Name                       Stmts   Miss  Cover
----------------------------------------------
todoman/__init__.py            3      0   100%
todoman/__main__.py            3      0   100%
todoman/cli.py               189     26    86%
todoman/configuration.py      47      7    85%
todoman/model.py             424     42    90%
todoman/radio.py              11     11     0%
todoman/ui.py                155     78    50%
todoman/version.py             1      0   100%
todoman/widgets.py            70     56    20%
----------------------------------------------
TOTAL                        903    220    76%


=================================================================== FAILURES ===================================================================
_______________________________________________________________ test_format_date _______________________________________________________________

    def test_format_date():
        """
        Tests the format_date function in todoman.ui.TodoFormatter
        """
        formatter = ui.TodoFormatter(DATE_FORMAT)
        today = datetime.now()
        tomorrow = today + timedelta(days=1)
        any_day = today + timedelta(days=randrange(2, 8))
    
        assert formatter.format_date("") == "        "
>       assert formatter.format_date(today) == "   Today"
E       assert '23-02-17' == '   Today'
E         - 23-02-17
E         +    Today

tests/test_formatter.py:19: AssertionError
=============================================== 1 failed, 46 passed, 3 xfailed in 48.83 seconds ================================================
ERROR: InvocationError: '/home/anubha/todoman/.tox/py3/bin/py.test --cov todoman'
___________________________________________________________________ summary ____________________________________________________________________
ERROR:   py3: commands failed
(venv_todo) anubha@anubha-Inspiron-3542:~/todoman/todoman$ 
```

**Comment by AnubhaAgrawal** [comment]
It show error in the issue that is already solved.

**Comment by AnubhaAgrawal** [comment]
even I pulled it from upstream but nothing is changed.

**Comment by untitaker** [comment]
@AnubhaAgrawal as said earlier, your pull request contains changes that you didn't intend to make. Please take a look at the "Files" tab on this pull request, you reverted some changes from another PR.

**Comment by hobarrera** [comment]
Superceded by #141

**Pull request closed without merging**